config/cluster-baremetal-operator/cluster-baremetal-operator.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     spec:
       containers:
       - command:
-        - /manager
+        - /cluster-baremetal-operator
         args:
         - --enable-leader-election
         image: controller:latest
```

Comment:
Doe the name in the follwoing line have to change too?

Reply:
Probably it's not very relevant as I'm not sure we're going to use this yaml to generate the deployment file, anyhow I'll revert it back

Reply:
Lets make a change to lines 27 and 31 to be consistent everywhere.